math/convertall/Portfile

```
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 ConvertAll 0.8.0 v
+github.tarball_from releases
+
+python.versions     34 35 36 37 38 39
```

```suggestion
python.versions     38 39
```

For now, I'd stick with 38 and 39. New versions can then be added on later.

See https://github.com/macports/macports-ports/pull/8620#issuecomment-703742167 for more info.